docs/changelog.md

```diff
@@ -0,0 +1,48 @@
+### 17-July-2020 - 18:54 CEST
```

Comment:
please add the latest entries from the wiki (it was updated today)

Reply:
Done!